<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowTitle">
   <string>Veyon Configurator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../veyon-configurator.qrc">
    <normaloff>:/resources/veyon-configurator.png</normaloff>:/resources/veyon-configurator.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="1">
     <widget class="QStackedWidget" name="configPages">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="GeneralConfigurationPage" name="generalConfigurationPage"/>
      <widget class="ServiceConfigurationPage" name="serviceConfigurationPage"/>
      <widget class="MasterConfigurationPage" name="masterConfigurationPage"/>
      <widget class="AccessControlPage" name="accessControlPage"/>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QListWidget" name="pageSelector">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="iconSize">
       <size>
        <width>48</width>
        <height>48</height>
       </size>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="viewMode">
       <enum>QListView::ListMode</enum>
      </property>
      <property name="uniformItemSizes">
       <bool>true</bool>
      </property>
      <property name="currentRow">
       <number>0</number>
      </property>
      <item>
       <property name="text">
        <string>General</string>
       </property>
       <property name="icon">
        <iconset resource="../veyon-configurator.qrc">
         <normaloff>:/resources/configure-shortcuts.png</normaloff>:/resources/configure-shortcuts.png</iconset>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Service</string>
       </property>
       <property name="icon">
        <iconset resource="../veyon-configurator.qrc">
         <normaloff>:/resources/application-x-sharedlib.png</normaloff>:/resources/application-x-sharedlib.png</iconset>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Master</string>
       </property>
       <property name="icon">
        <iconset resource="../veyon-configurator.qrc">
         <normaloff>:/resources/application-x-ms-dos-executable.png</normaloff>:/resources/application-x-ms-dos-executable.png</iconset>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Access control</string>
       </property>
       <property name="icon">
        <iconset resource="../veyon-configurator.qrc">
         <normaloff>:/resources/network-vpn.png</normaloff>:/resources/network-vpn.png</iconset>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="standardButtons">
       <set>QDialogButtonBox::Apply|QDialogButtonBox::Reset</set>
      </property>
      <property name="centerButtons">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>572</width>
     <height>36</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionLoadSettings"/>
    <addaction name="actionSaveSettings"/>
    <addaction name="separator"/>
    <addaction name="actionResetConfiguration"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAboutVeyon"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Help"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionSaveSettings">
   <property name="icon">
    <iconset resource="../../core/core.qrc">
     <normaloff>:/resources/document-save.png</normaloff>:/resources/document-save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save settings to file</string>
   </property>
   <property name="toolTip">
    <string>Save settings to file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionLoadSettings">
   <property name="icon">
    <iconset resource="../../core/core.qrc">
     <normaloff>:/resources/document-open.png</normaloff>:/resources/document-open.png</iconset>
   </property>
   <property name="text">
    <string>L&amp;oad settings from file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionAboutVeyon">
   <property name="icon">
    <iconset resource="../../core/core.qrc">
     <normaloff>:/resources/help-about.png</normaloff>:/resources/help-about.png</iconset>
   </property>
   <property name="text">
    <string>About Veyon</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionResetConfiguration">
   <property name="icon">
    <iconset resource="../../core/core.qrc">
     <normaloff>:/resources/edit-delete.png</normaloff>:/resources/edit-delete.png</iconset>
   </property>
   <property name="text">
    <string>Reset configuration</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AccessControlPage</class>
   <extends>QWidget</extends>
   <header>AccessControlPage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GeneralConfigurationPage</class>
   <extends>QWidget</extends>
   <header>GeneralConfigurationPage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MasterConfigurationPage</class>
   <extends>QWidget</extends>
   <header>MasterConfigurationPage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ServiceConfigurationPage</class>
   <extends>QWidget</extends>
   <header>ServiceConfigurationPage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../core/core.qrc"/>
  <include location="../veyon-configurator.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pageSelector</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>configPages</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>502</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAboutVeyon</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>aboutVeyon()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionResetConfiguration</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>resetConfiguration()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>680</x>
     <y>426</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>aboutVeyon()</slot>
  <slot>resetConfiguration()</slot>
 </slots>
</ui>
